src/client-scopes/routes/Mapper.ts

```diff
@@ -7,10 +7,11 @@ export type MapperParams = {
   realm: string;
   id: string;
   mapperId: string;
+  type: string;
```

Comment:
Let's keep the order in the interface the same as the path.
```suggestion
  id: string;
  type: string;
  mapperId: string;
```